packages/server/scripts/integrations/postgres/reset.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 docker-compose down
 docker volume prune -f
+docker volume rm postgres_pg_data
```

Comment:
You could instead run `docker-compose down -v` to delete all volumes from the compose YAML.

Reply:
Wasn't aware of this - nice!